crates/hc_bundle/src/packing.rs

```diff
@@ -4,10 +4,28 @@
 
 use crate::error::{HcBundleError, HcBundleResult};
 use holochain_util::ffs;
+use holochain_wasmer_host::prelude::*;
 use mr_bundle::RawBundle;
 use mr_bundle::{Bundle, Manifest};
 use std::path::Path;
 use std::path::PathBuf;
+use std::str::FromStr;
+use std::sync::Arc;
+use wasmer_middlewares::Metering;
+
+const WASM_METERING_LIMIT: u64 = 100_000_000_000;
+
+// TODO-connor: don't have this code copy-pasted from real_ribosome.
+// instead place it in TBD appropriate place to be shared between?
+pub fn cranelift() -> Cranelift {
+    let cost_function = |_operator: &wasmparser::Operator| -> u64 { 1 };
+    // @todo 100 giga-ops is totally arbitrary cutoff so we probably
+    // want to make the limit configurable somehow.
+    let metering = Arc::new(Metering::new(WASM_METERING_LIMIT, cost_function));
+    let mut cranelift = Cranelift::default();
+    cranelift.canonicalize_nans(true).push_middleware(metering);
+    cranelift
+}
```

Comment:
@Connoropolous it looks like you could just move this, and `WASM_METERING_LIMIT`, from `holochain` to `holochain_types`, which is already a depdendency of `hc_bundle`